index.html

```diff
@@ -4633,26 +4633,112 @@ <h2>Key and Signature Expiration</h2>
-<a href="#did-controller"></a>. Not all <a>DID method</a>
-specifications will recognize control from <a>DIDs</a> registered using other
-<a>DID methods</a> and they might restrict third-party control to <a>DIDs</a>
-that use the same method. Access control and key recovery in a
-<a>DID method</a> specification can also include a time lock feature to
-protect against key compromise by maintaining a second track of control for
-recovery.
+Verification method revocation is a reactive security measure.
+      </p>
+
+      <p>
+Verification method revocation applies only to the current or latest
+version of a DID Document.
+      </p>
+  
+      <p>
+        If a verification method is no longer exclusively accessible to the
+        controller or parties trusted to act on behalf of the controller, it
+        should be revoked immediately to reduce the risk of masquerading, theft
+        and fraud.
+      </p>
+  
+      <p class="advisement">
+It is considered a best practice to support key revocation.
+      </p>
+  
+      <p class="advisement">
+A controller should immediately revoke any verification method that is believed to
+be compromised.
+      </p>
+  
+      <p class="advisement">
+Verifiers may choose to accept or reject proofs or signatures originating 
+from revoked verification methods, at their own discretion.
+      </p>
+  
+      <p class="note">
+As described in Section <a href="#verification-method-types"></a>,
+absence of a verification method is the only form of revocation that
+applies to all DID Methods.
+      </p>
+  
+      <p>
+Section <a href="#method-operations"></a> specifies the <a>DID</a>
+operations to be supported by a <a>DID method</a> specification,
+including <a href="#update">update</a> and <a href="#deactivate">
+deactivate</a> which MAY be used to remove verification method from
+a DID Document.
+      </p>
+
+      <p>
+Not all DID Methods support verification method revocation.
+      </p>
+  
+      <p>
+Even if a verification method is present in a DID Document, additional
+information, such as a public key revocation certificate, or an external
+allow or deny list, might be used to determine whether a verification
+method has been revoked.
+      </p>
+  
+      <p class="note">
+Compromise of a verification method allows the attacker access to the
+verification relationship of the controller, for example authentication.
+The attacker might not be indistinguishable from the legitimate
+controller from the time the compromised verification method was
+registered, to the time it was revoked.
+      </p>
+
+      <p class="note">
+The day-to-day operation of any software relying on a compromised verification
+method, such as a user’s operating system, antivirus, or endpoint protection 
+software, might be impacted when the verification method is publicly revoked.
+      </p>
+
+<p class="note">
+Although verifiers may choose not to accept proofs or signatures from revoked verification method, 
```

Comment:
```suggestion
Although verifiers may choose not to accept proofs or signatures from a revoked verification method, 
```

Reply:
Favoring different line change that changes may -> might.

Reply:
I have updated @TallTed's edit to change may->might